assets/src/blocks/Covers/CoversScript.js

```diff
@@ -0,0 +1,8 @@
+import { CoversFrontend } from './CoversFrontend';
+
+document.querySelectorAll( `[data-render='planet4-blocks/covers']` ).forEach(
```

Comment:
Also has the advantage that we can migrate `planet4-blocks/covers-beta` to `planet4-blocks/covers` without changing the block's inner content.